atom/browser/browser.h

```diff
@@ -223,9 +225,7 @@ class Browser : public WindowListObserver {
   std::string version_override_;
   std::string name_override_;
 
-#if defined(OS_LINUX)
   int current_badge_count_ = 0;
```

Comment:
I think this could just be `badge_count_`, the `current` prefix is kind of redundant.